features/collections/collectible.feature

```diff
@@ -112,3 +112,70 @@ Feature: Collectible items
     Then I should see "Bookmarks (0)"
       And I should not see "Tundra penguins"
 
+  Scenario: Bookmarks of deleted items are included on the collection's Manage
```

Comment:
Alas, one of the scenarios needs to be moderated, so it'll take a bit more work to make the collection moderated after the Background step. I can do that in a separate pull request and refactor the existing tests in this as well, since I think they can also be reworked to share the same background.